### ISSUE TITLE
web socket ssl proxypass support

### ISSUE THREAD

**Issue by lukasmrtvy** [abricot/webinterface.arch#16]
Hi,
Is possible to modify explicitly hardcoded jsonrpc url paths from http://  to http(s):// ? 

Even if you set proxy ( ws to wss) , 
client will always call http://json, then will come error notification about loading non ssl content from ssl site (mixed content: chrome)
Thanks

**Comment by nicolasabric** [comment]
@muhahacz agreed. I will add the possibility to choose between secure and non-secure scheme in the settings.

**Comment by lukasmrtvy** [comment]
@nicolasabric 
No need to choose,...
https://github.com/jez500/chorus2/issues/54

**Comment by lukasmrtvy** [comment]
@nicolasabric ?
